### ISSUE TITLE
Mouse cursor always visible in macOS

### ISSUE THREAD

**Issue by girlsperhour** [HarbourMasters/2ship2harkinian#750]
Wanted to know if there was a way to hide the cursor. It seems to remain visible no matter what I do.

**Comment by Archez** [comment]
Hi, currently the only way that the mouse cursor hides itself is if you fullscreen the program and then hide the F1 menu bar.

I will note that in this case, you have to use the fullscreen from the menu bar 2Ship drop down, not the green window fullscreen button from the OS.

I'll raise an issue about the two different fullscreens not working the same on the upstream repo that manages this (LUS)

**Comment by Malkierian** [comment]
On a somewhat related note, if you go fullscreen with the menubar visible, then turn the menubar off in the menubar itself, the cursor doesn't hide. Have to toggle the menubar with F1. Most of this is resolved with the modern menu, as I added an option to always show the cursor when fullscreen with the menu closed, and otherwise it's just always on. It's a little simpler flow. Not sure whether that would show the same issue as the menubar cursor visibility stuff does.

**Comment by girlsperhour** [comment]
Thank you! I don't like to use F11 to fullscreen the game because it extends the game up into the MacBook notch. Fullscreening with the green traffic light keeps it under the (OS's) menubar.

**Comment by Archez** [comment]
> Thank you! I don't like to use F11 to fullscreen the game because it extends the game up into the MacBook notch. Fullscreening with the green traffic light keeps it under the (OS's) menubar.

Yeah this is what I was observing, so I'll track that against LUS and hopefully that will make its way back to here in the future.
